mesheryctl/pkg/utils/helpers.go

```diff
@@ -186,8 +186,17 @@ func PreReqCheck(subcommand string) error {
 }
 
 func startdockerdaemon(subcommand string) error {
+	// read user input on whether to start Docker daemon or not.
+	var userinput string
+	fmt.Printf("Do you want to start Docker daemon now?(y/n): ")
+	fmt.Scan(&userinput)
+	userinput = strings.TrimSpace(userinput)
+	userinput = strings.ToLower(userinput)
+	if userinput == "n" || userinput == "no" {
+		return errors.Errorf("please start Docker then run the command `mesheryctl system %s`", subcommand)
```

Comment:
@hexxdump, we may be able to exclude this message as it is duplicative of another message. Example here:

```
➜  mesheryctl git:(hexxdump/master) ✗ ./mesheryctl system start
Docker daemon is not running
Do you want to start Docker daemon now?(y/n): n
Error: failed to start Docker daemon. Run `mesheryctl system start` once docker is started.: please start Docker then run the command `mesheryctl system start`
Usage:
  mesheryctl system start [flags]

Flags:
  -h, --help          help for start
      --reset         (optional) reset Meshery's configuration file to default settings.
      --skip-update   (optional) skip checking for new Meshery's container images.

Global Flags:
      --config string              config file (default location is: /Users/lee/.meshery/meshery.yaml)
      --mesheryctl-config string   mesheryctl config file to override defaults (default file: <unset>)
```

Reply:
Agreed :+1:

Reply:
Fixed the log in the PreReqCheck.

Reply:
New format:

```
➜  mesheryctl git:(hexxdump/master) ✗ ./mesheryctl system start
Docker daemon is not running
Do you want to start Docker daemon now?(y/n): y
Attempting to start Docker daemon
Error: failed to start Docker daemon.: please start Docker then run the command `mesheryctl system start`: exit status 1
Usage:
  mesheryctl system start [flags]

Flags:
  -h, --help          help for start
      --reset         (optional) reset Meshery's configuration file to default settings.
      --skip-update   (optional) skip checking for new Meshery's container images.

Global Flags:
      --config string              config file (default location is: /Users/lee/.meshery/meshery.yaml)
      --mesheryctl-config string   mesheryctl config file to override defaults (default file: <unset>)
```
An improvement. 👍 Ideally, this will read: "Failed to start Docker. Please start Docker, then run the command `mesheryctl system start`."